tick/survival/convolutional_sccs.py

```diff
@@ -753,7 +753,7 @@ def _construct_solver_obj(step, max_iter, tol, print_every, record_every,
         # seed cannot be None in SVRG
         solver_obj = SVRG(step=step, max_iter=max_iter, tol=tol,
                           print_every=print_every, record_every=record_every,
-                          verbose=verbose, seed=seed)
+                          verbose=verbose, seed=seed, n_threads=8)
```

Comment:
Why is `n_threads=8` hardcoded here? Maybe use a class attribute to allow the user to change this value.

Btw, Stéphane told me it'd be nice to add the option to use Barzilai Borwein rule, `SVRG(..., step_type='bb')`, which allows running the solver without having to compute the lip const when it's too expensive. It should be user's choice to use either step computed from lip const or using `bb` rule